ofxdump/Makefile.am

```diff
@@ -1,6 +1,6 @@
 bin_PROGRAMS = ofxdump
 ofxdump_LDADD = $(top_builddir)/lib/libofx.la
-ofxdump_SOURCES = cmdline.h cmdline.c ofxdump.cpp
+ofxdump_SOURCES = cmdline.h cmdline.c ofxdump.cpp $(top_builddir)/lib/getopt.c $(top_builddir)/lib/getopt1.c
```

Comment:
This change does not make sense to me - in this PR there is neither a file getopt.c introduced in builddir, nor one named getopt1.c.